### PR TITLE
fix vector directions

### LINKED ISSUE #11: Calculate volume and bounding box
not considered min or max can be a negative number.
if (max x < 0 || min x > 0) {
  return max x - min x
}
return Math.abs(min x) + max x
in stead of max x - min x

Also, the volume value calculated by node stl is different with cura or admesh result! and the returned value is not reasonable in some cases

### PR DESCRIPTION
**Pull request by alra7il**
Fixes #11

### CODE REVIEW

**Comment by johannesboyne** [comment]
This does not seem to work, the tests won't run. I'll setup a CI in order to make this easier to spot.

File: index.js [hunk 1]
```diff
@@ -155,8 +155,10 @@ function parseSTLString(stl) {
 				let vector = vertex
 					.replace('vertex', '')
 					.match(/[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?/g);
-
-				triangle[i] = new Vector3(vector[0], vector[1], vector[2]);
+        const x = vector[0] * Math.pow(10, Number(vector[1]))
+        const y = vector[2] * Math.pow(10, Number(vector[3]))
+        const z = vector[4] * Math.pow(10, Number(vector[5]))
```
Comment (johannesboyne) [inline index.js]: The vector array only has three items `vector[0..2]`

Comment (alra7il) [reply index.js]: vector [ '6.520870', '-001', '-1.517670', '+001', '-7.011640', '+000' ]
vertex  6.520870e-001 -1.517670e+001 -7.011640e+000
You set second item of vector array ('-001') to vector3 . But It isn't correct number! its power of 10. see vertex first item : 6.520870e-001
With these changes the result is correct and is the same as Cura Engine and Sergei burkov plugin result.

Comment (johannesboyne) [reply index.js]: I looked at the vector in the debugger, and for the test files it looks like this:
```
// ...
[ '-1.995', '-41.6889', '2.5' ]
// ...
```

thus three.

If you create the PR again, you'll see the CI output and the failing tests

Comment (johannesboyne) [reply index.js]: https://github.com/johannesboyne/node-stl/pull/14

**Pull request merged**